sidecar/Makefile

```diff
@@ -14,6 +14,10 @@ all: build push
 build:
 	docker build --tag $(TAG) --file ${root_dir}sidecar/Dockerfile ${root_dir}sidecar/
 
+build:
```

Comment:
This duplicates the target above

Reply:
Thanks! I didn't add this change to the commit, now it is corrected.